src/main/resources/assets/arcanuscontinuum/lang/fr_fr.json

```
@@ -0,0 +1,334 @@
+	"item.arcanuscontinuum.battle_mage_chestplate_waxed_exposed": "Plastron de Mage de Guerre Erodé Ciré",
+	"item.arcanuscontinuum.battle_mage_leggings_waxed_exposed": "Jambières de Mage de Guerre Erodées Cirées",
+	"item.arcanuscontinuum.battle_mage_boots_waxed_exposed": "Bottes de Mage de Guerre Erodées Cirées",
+	"item.arcanuscontinuum.battle_mage_helmet_waxed_weathered": "Casque de Mage de Guerre Erodé Ciré",
+	"item.arcanuscontinuum.battle_mage_chestplate_waxed_weathered": "Plastron de Mage de Guerre Erodé Ciré",
+	"item.arcanuscontinuum.battle_mage_leggings_waxed_weathered": "Jambières de Mage de Guerre Erodées Cirées",
+	"item.arcanuscontinuum.battle_mage_boots_waxed_weathered": "Bottes de Mage de Guerre Erodées Cirées",
+	"item.arcanuscontinuum.battle_mage_helmet_waxed_oxidized": "Casque de Mage de Guerre Oxydé Ciré",
+	"item.arcanuscontinuum.battle_mage_chestplate_waxed_oxidized": "Plastron de Mage de Guerre Oxydé Ciré",
+	"item.arcanuscontinuum.battle_mage_leggings_waxed_oxidized": "Jambières de Mage de Guerre Oxydées Ciré",
+	"item.arcanuscontinuum.battle_mage_boots_waxed_oxidized": "Bottes de Mage de Guerre Oxydées Ciré",
+	"item.arcanuscontinuum.battle_mage_upgrade_smithing_template": "Modèle de Forge",
+	"upgrade.arcanuscontinuum.battle_mage_upgrade": "Amélioration en Mage de Guerre",
+	"item.arcanuscontinuum.smithing_template.battle_mage_upgrade.applies_to": "Armure en Diamand",
+	"item.arcanuscontinuum.smithing_template.battle_mage_upgrade.ingredients": "Éclats d'Améthyste",
+	"item.arcanuscontinuum.smithing_template.battle_mage_upgrade.base_slot_description": "Ajouter une Armure en Diamand",
+	"item.arcanuscontinuum.smithing_template.battle_mage_upgrade.additions_slot_description": "Ajouter un Éclat d'Améthyste",
+	"item.arcanuscontinuum.spell_book": "Grimoire de Sorts",
+	"item.arcanuscontinuum.scroll_of_knowledge": "Parchemin de Savoir",
+	"item.arcanuscontinuum.wizard_spawn_egg": "Oeuf d'Apparition de Sorcier",
+	"item.arcanuscontinuum.opossum_spawn_egg": "Oeuf d'Apparition d'Oppossum",
+
+	"block.arcanuscontinuum.magic_door": "Porte Magique",
+	"block.arcanuscontinuum.arcane_workbench": "Etabli Arcanique",
+	"block.arcanuscontinuum.magic_block": "Bloc Magique",
+	"block.arcanuscontinuum.unbreakable_magic_block": "Bloc Magique",
+
+	"entity.arcanuscontinuum.wizard": "Sorcier",
+	"entity.arcanuscontinuum.opossum": "Opossum",
+	"entity.arcanuscontinuum.necro_skeleton": "Necro Squellette",
+	"entity.arcanuscontinuum.mana_shield": "Bouclier de Mana",
+	"entity.arcanuscontinuum.magic_projectile": "Projectile Magique",
+	"entity.arcanuscontinuum.magic_rune": "Rune Magique",
+	"entity.arcanuscontinuum.area_of_effect": "Zone d'Effet",
+	"entity.arcanuscontinuum.smite": "Rayon Celeste",
+	"entity.arcanuscontinuum.beam": "Rayon",
+	"entity.arcanuscontinuum.guardian_orb": "Orbe du Guardian",
+	"entity.arcanuscontinuum.aggressorb": "Aggressorbe",
```

```suggestion
	"entity.arcanuscontinuum.aggressorb": "Agressorbe",
```
Only one 'g' to "agression" in French